test/ASTGen/decls.swift

```diff
@@ -9,12 +9,11 @@
 
 // RUN: %diff -u %t/astgen.ast %t/cpp-parser.ast
 
-// RUN: %target-run-simple-swift(-Xfrontend -disable-availability-checking -enable-experimental-feature SwiftParser -enable-experimental-feature ParserASTGen -enable-experimental-feature ValueGenerics)
+// RUN: %target-run-simple-swift(-Xfrontend -disable-availability-checking -enable-experimental-feature ParserASTGen -enable-experimental-feature ValueGenerics)
 
 // REQUIRES: executable_test
 // REQUIRES: swift_swift_parser
 // REQUIRES: swift_feature_ParserASTGen
-// REQUIRES: swift_feature_SwiftParser
```

Comment:
These tests were unexpectedly disabled. `SwiftParser` doesn't exist (anymore?) but the command still had `-enable-experimental-feature SwiftParser`. cc: @drodriguez

Reply:
I will have a look at what can be done.

Reply:
#77542

Found this one and a couple more. Seems that `SwiftParser` was renamed to `ParserASTGen`, but the old spelling was kept in the tests (together with the new spelling).